persist-cli/src/main/java/io/ballerina/persist/nodegenerator/BalSyntaxGenerator.java

```diff
@@ -492,7 +493,7 @@ private static Node generateMetadataRecord(Module entityModule) {
             }
             entityMetaData.append(String.format(METADATARECORD_KEY_FIELD_TEMPLATE, keyFields));
             mapBuilder.append(String.format(METADATARECORD_ELEMENT_TEMPLATE,
-                    entity.getResourceName(), entityMetaData));
+                    entity.getEntityName().toLowerCase(Locale.ENGLISH), entityMetaData));
```

Comment:
Metadata maps can also use entity resources as identifiers. right? 
why do we need only lowercase to the key

Reply:
Yes. 
Updated in [757e5da](https://github.com/ballerina-platform/persist-tools/pull/132/commits/757e5da60926fdb274a66fbe61ea1d7cf3d6cce3)

Reply:
Sorry, I have to change my mind again. it looks better to keep entity names as it is for the metadata. make only resource names plural. and also better we get this from the Entity class as another attribute like `metaKey`